lib/vsc/utils/fancylogger.py

```diff
@@ -512,7 +543,8 @@ def logToUDP(hostname, port=5005, enable=True, datagramhandler=None, name=None):
                            )
 
 
-def _logToSomething(handlerclass, handleropts, loggeroption, enable=True, name=None, handler=None):
+def _logToSomething(handlerclass, handleropts, loggeroption,
+                    enable=True, name=None, handler=None, formatterclass=logging.Formatter):
```

Comment:
i would have to check, but the default formatterclass should be `formatterclass=None`, and then in the method
```python
if formatterclass is None:
    formatterclass=logging.Formatter
```